agent/fw_mediawiki.c

```diff
@@ -24,6 +24,14 @@
  * done by trapping ApiMain::__construct. This takes as its first argument a
  * WebRequest object. That object has an array called 'data'. That array will
  * contain a member named 'action'.
+ *
+ *
+ * txn naming scheme:
+ * In this case, `nr_txn_set_path` is called before `NR_PHP_WRAPPER_CALL` with
```

Comment:
This says it is called before but below it is not setup with the "before after clean" wrapper function - is this a problem?